scss/tools.scss

```diff
@@ -0,0 +1,14 @@
+.tools-section {
+  display: flex;
+  flex-direction: column;
+  margin: 40px auto; 
+  max-width: 1000px; 
+  width: 100%;
+
+  .techs {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-start;
```

Comment:
This is the default value, check this [page](https://www.w3schools.com/cssref/css3_pr_justify-content.asp), so delete it, please